tests/utils/image_utils_tests.py

```diff
@@ -69,9 +69,10 @@ def test_rescale_volume():
     nose.tools.assert_tuple_equal(res_volume.shape, (10, 34, 53))
     # downsample isotropically, 0.5 downsampling
     res_volume = image_utils.rescale_nd_image(uni_thr_tst_image, 0.7)
```

Comment:
sorry, this was a typo, not sure how I missed this. Please pass nd_image instead of uni_thr_tst_image and the res_volume shape should be 3, 22, 22

Reply:
I changed it to nd_image and the res_volume.shape ended up being (4, 22, 22). I hope that's ok!